builtin/client/init.lua

```diff
@@ -6,8 +6,18 @@ local commonpath = scriptpath.."common"..DIR_DELIM
 dofile(clientpath .. "register.lua")
 dofile(commonpath .. "after.lua")
 dofile(commonpath .. "chatcommands.lua")
+dofile(clientpath .. "chatcommands.lua")
```

Comment:
hmmm common has registration, and here we need client side definitions but it seems this PR doesn't contain chatcommands.lua, missing chatcommands.lua @red-001  ?